<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QMainWindow" name="sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
color: rgb(203, 203, 203)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	
	
	font: 700 14pt &quot;Trebuchet MS&quot;;
}</string>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	
	
	font: 700 14pt &quot;Trebuchet MS&quot;;
}</string>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	
	
	font: 700 14pt &quot;Trebuchet MS&quot;;
}</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	font: 700 20pt &quot;Comic Sans MS&quot;;
	color: rgb(0, 0, 0);

}</string>
         </property>
         <property name="text">
          <string>SignUp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="styleSheet">
          <string notr="true">QLineEdit{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
            </property>
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
